src/test/java/com/wikia/webdriver/testcases/mediatests/lightboxtests/LightboxTests.java

```diff
@@ -142,10 +149,11 @@ public void LightboxTest_008_filepage_video() {
     filePage.verifyVideoAutoplay(true);
   }
 
-  @Test(groups = {"LightboxTest", "LightboxTest_009", "Media"})
   /**
    * Open lightbox from Special:NewFiles, verify title url, verify More Info button and verify file page (logged-in user)
    */
+  @Test(groups = {"LightboxTest", "LightboxTest_009", "Media"})
```

Comment:
Please move groups `{"LightboxTest", "Media"}`` from all methods in this file above class